sphinx/source/members/adding_member.rst

```diff
@@ -3,25 +3,27 @@ Adding New Members
 
 It is possible for existing members to add new members to the consortium after a CCF network has been started.
 
+.. note:: The maximum number of active consortium members at any given time is 255.
```

Comment:
Not closing https://github.com/microsoft/CCF/issues/851 just yet. I will add some checks in CCF to make sure that the 256th member `ACK` fails.